include/quicr/publish_track_handler.h

```diff
@@ -97,8 +97,9 @@ namespace quicr {
                     break;
                 case TrackMode::kStream:
                     // TODO: Is a default acceptable, or enforce?
-                    stream_mode_ = stream_mode.has_value() ? stream_mode.value()
-                                                           : messages::StreamHeaderType::kSubgroupZeroWithExtensions;
+                    stream_mode_ = stream_mode.has_value()
+                                     ? stream_mode.value()
+                                     : messages::StreamHeaderType::kSubgroup0NotEndOfGroupWithExtensions;
```

Comment:
This is totally fine, though could also be achieved with
```suggestion
                    if (!stream_mode.has_value()) {
                        stream_mode = messages::StreamHeaderType::kSubgroup0NotEndOfGroupWithExtensions;
                    }
```
Though it does introduce tabbing, so as it is is great too.